source/contributing/documentation.md

```diff
@@ -5,6 +5,45 @@ Documentation contributions should follow to the [writing style recommendations]
 
 Feel free to in touch with the [Nix documentation team](https://nixos.org/community/teams/documentation) if you want to help out.
 
+## Documentation framework
+We aim to build our documentation according to the [Diataxis][diataxis] framework, which divides documentation into four categories:
+
+- Reference
+- How-to
+- Explanation
+- Tutorial
+
+We've found that contributors struggle to know the difference between these categories, while we _strongly_ recommend reading up on the Diataxis framework, we can summarize it in this way:
+
+- Reference
+  - Reference matieral includes technical documentation, API documentation, etc.
+  - Reference material should be relatively terse, simply listing which functions/classes/etc exist, but should still clearly define how to use the code and how it relates to the rest of the ecosystem.
+- How-to (we call these "Recipes")
+  - A recipe is a list of steps to achieve a specific goal.
+  - The goal is not to show how or why something works, but instead to show how to solve a specific problem.
+  - A Recipe assumes that the reader already has the background to understand the topic at hand and therefore doesn't need to explain the introduction of each new concept.
+- Explanation (we call these "Concepts")
+  - A Concept describes the internals of a piece of code or how to think about a particular concept or entity in the ecosystem.
+  - A Concept can also describe the historical context behind why something works the way that it does today.
+  - If you find yourself wanting to write about the nitty gritty details of how something works, you most likely want to write an Explanation.
+- Tutorial
+  - A tutorial walks the user through a particular activity to help give them muscle memory with accepted tools and patterns in the ecosystem.
+  - While the activity itself is important, the goal is to get the reader familiar with the patterns and tools in the ecosystem.
+
+We also find that people struggle with the difference between a How-to/Recipe and a Tutorial.
+One analogy is landing an airplane.
```

Comment:
I don't think the analogy is suitable. To the contrary, recipes are more like checklists for experienced pilots (or surgeons for that matter) who already know the details and need a high level guide to perform a task that consists of multiple well-known subroutines that need no additional explanation. There may even be conditionals, depending on the situation.

A tutorial on the other hand would tell a novice where each button in the plane (or each blood vessel in the operated region) is located while instructing the learner to perform every granular step in succession such that there is no ambiguity and no opportunity for failure, i.e. no decisions to be made on the learner's end.